components/worker/internal/bootstrap/config_fetcher.go

```
@@ -0,0 +1,279 @@
+		S3Region:          cfg.ObjectStorageRegion,
+		S3AccessKeyID:     cfg.ObjectStorageAccessKeyID,
+		S3SecretAccessKey: cfg.ObjectStorageSecretKey,
+		S3UsePathStyle:    cfg.ObjectStorageUsePathStyle,
+		S3DisableSSL:      cfg.ObjectStorageDisableSSL,
+	})
+	if err != nil {
+		return nil, fmt.Errorf("initializing fetcher data storage: %w", err)
+	}
+
+	deps.service.FetcherDataStorage = storage.NewFetcherStorageAdapter(fetcherStorageClient)
+	logger.Log(ctx, clog.LevelInfo, "Fetcher data storage configured (S3)",
```

golangci-lint (2.11.4)

[error] missing whitespace above this line (no shared variables above expr)

(wsl_v5)